config.xml

```diff
@@ -89,4 +89,5 @@
         <variable name="UPDATE_METHOD" value="auto" />
     </plugin>
     <engine name="android" spec="7.0.0" />
+    <plugin name="cordova-sqlite-storage" spec="3.1.0" />
```

Comment:
why use sqlite for something as tiny as the language selection? i think localstorage works well too, dont u think?

Reply:
https://ionicframework.com/docs/building/storage - following this guide. I think better to use native stuff

Reply:
ok good point